doc/flatpak-metadata.xml

```diff
@@ -721,12 +721,13 @@
             </para>
             <para>
                 The default policy for the session bus only allows the
-                application to own its own application ID and
-                subnames. For instance if the app is called
-                "org.my.App", it can only own "org.my.App" and
-                "org.my.App.*". Its also only allowed to talk to the
-                bus itself (org.freedesktop.DBus) and the portal APIs
-                APIs (bus names of the form org.freedesktop.portal.*).
+                application to own its own application ID, its
+                subnames and its own application id as a subname of 
+                "org.mpris.MediaPlayer2". For instance if the app is called
+                "org.my.App", it can only own "org.my.App", "org.my.App.*"
+                and "org.mpris.MediaPlayer2.org.my.App". Its also only
+                allowed to talk to the bus itself (org.freedesktop.DBus) 
+                and the portal APIs (bus names of the form org.freedesktop.portal.*).
```

Comment:
It's a bit misleading to say that the app can only talk to the bus and portals, because the `own` permission implies `talk`, so it can also talk to for example `org.my.App.OtherApp`. While you're changing this text anyway, perhaps this:

```suggestion
                and "org.mpris.MediaPlayer2.org.my.App".
                It is only allowed to talk to names matching those patterns, plus
                the bus itself (org.freedesktop.DBus) 
                and the portal APIs (bus names of the form org.freedesktop.portal.*).
```

Reply:
Thanks, makes sense